### PR TITLE
Test support for bin deployed aspnetcore.dll

### PR DESCRIPTION
**Pull request by Tratcher**
https://github.com/aspnet/IISIntegration/issues/292 @JunTaoLuo @pranavkm 
// CC: @shirhatti @pan-wang

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs [hunk 1]
```diff
@@ -57,8 +68,13 @@ private CancellationToken StartIISExpress(Uri uri, string contentRoot)
                 // Pass on the applicationhost.config to iis express. With this don't need to pass in the /path /port switches as they are in the applicationHost.config
                 // We take a copy of the original specified applicationHost.Config to prevent modifying the one in the repo.
 
+                // The nupkg build of ANCM does not support Win7. https://github.com/aspnet/AspNetCoreModule/issues/40.
+                // If Win7 were supported we could just hardcode "aspnetcore.dll" in the config files.
+                var ancmPath = IsWin8orLater ? "aspnetcore.dll" : @"%ProgramFiles%\IIS Express\aspnetcore.dll";
```
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs]: So we are skipping Win2008 R2 as well?

Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs]: Correct, it has the same issue and version.

Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs]: Also, the old path looks like it's `%SystemRoot%\System32\inetsrv\aspnetcore.dll` why are we changing that now?

Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs]: Yes. When we first wrote these tests there was only an installer for IIS so we used that dll. Now there's also an installer for express.

**Comment by JunTaoLuo** [comment]
Changes look good. :shipit: Though @shirhatti had a question about 32-bit.

**Comment by Tratcher** [comment]
Updated to use an absolute path for the bin deployed aspnetcore.dll. I realized the relative reference was ambiguous and preferred the dll adjacent to IISExpress.exe.

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/project.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Testing": "1.1.0-*",
+    "Microsoft.DotNet.PlatformAbstractions": "1.1.0-*",
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/project.json]: Can we use https://github.com/aspnet/Common/blob/dev/src/Microsoft.Extensions.RuntimeEnvironment.Sources/RuntimeEnvironment.cs instead?

**Comment by pranavkm** [comment]
:shipit:

**Comment by Tratcher** [comment]
FYI: This is going to be rebased on rel/1.1.0 and then merged back to dev. It will also be cherry-picked down to LTS (rel/1.0.*)

**Comment by Tratcher** [comment]
Replacing this PR with one targeting rel/1.1.0

**Pull request closed without merging**